src/sidebar/search-client.js

```diff
@@ -1,5 +1,9 @@
 import { TinyEmitter } from 'tiny-emitter';
 
+/**
```

Comment:
As you can see here and a bit below, I started down the path of adding TSdoc here, but I didn't want to get too hung up on it lest it cause delay or distraction. I know that this module isn't included in type checking yet, anyway, but wanted to a little bit...

Reply:
Hi Lyza - This particular file is actually checked because it is imported by a file that is checked. I don't think I mentioned that transitive aspect of the way it works. All the JSDoc changes here typecheck just fine though 🙂.  To see this in action:

1. Run `yarn typecheck` on this branch - no errors
2. Change `[options.chunkSize]` to `[options.pageSize]` in the JSDoc or make some other kind of "mistake", then run `yarn typecheck` and you'll get an error.

Reply:
Oh, yes, I know! I did test my work :).